packages/markdown-docx/src/CiceroMarkToOOXMLRules.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { sanitizeHtmlChars } = require('./CiceroMarkToOOXMLHelpers');
+
+/**
+ * Inserts text.
+ *
+ * @param {string}  value     Text to be rendered
```

Comment:
You are not rendering anything. This was true for the add-in but not markdown transform. Better wording would be `enclosing value of the OOXML tag`